docs/install/containerd/containerd-install.md

```diff
@@ -0,0 +1,99 @@
+# Install Kata Containers with containerd
+
+1. Perform initial checks
+
+   Check to see if Kata Containers is already installed:
+
+   ```bash
+   $ command -v kata-runtime &>/dev/null && { echo >&2 "ERROR: Please remove existing Kata Containers installation"; exit 1; }
+   ```
+
+1. Install dependencies
+
+   ```bash
+   $ source /etc/os-release || source /usr/lib/os-release
+   $ packages='containerd curl git jq'
+   $ case "$ID" in
+   $   ubuntu) sudo apt-get -y install $packages ;;
+   $   *) echo >&2 "ERROR: Unsupported distro: $ID"; exit 1 ;;
+   $ esac
+   ```
+
+1. Install Kata Containers
+
+   ```bash
+   $ tmpdir=$(mktemp -d)
+   $ pushd "$tmpdir" &>/dev/null
+   $ url="https://api.github.com/repos/kata-containers/runtime/releases/latest"
+   $ full_url=$(curl -sL "$url" | jq -r '.assets[0].browser_download_url')
+   $ arch=$(uname -m)
+   $ echo "$full_url" | grep -q "$arch" || { echo >&2 "ERROR: No Kata Containers package for $arch architecture"; exit 1; }
+   $ curl -LO "$full_url"
+   $ file=$(echo "$full_url" | awk -F\/ '{print $NF}')
+   $ sudo tar -C / -xvf "./${file}"
+   $ popd &>/dev/null
+   ```
+
+   Show version details:
+
+   ```bash
+   $ export PATH=$PATH:/opt/kata/bin
+   $ kata-runtime --version
+   ```
+
+1. Configure containerd
+
+   Stop the service to allow the configuration to be modified:
+
+   ```bash
+   $ sudo systemctl stop containerd
+   ```
+
+   Backup the original containerd configuration:
+
+   ```bash
+   $ cfg="/etc/containerd/config.toml"
+   $ sudo grep -q "io.containerd.kata.v2" "$cfg" || sudo cp "$cfg" "${cfg}.ORIGINAL"
+   ```
+
+   Add the Kata Containers configuration details:
+
+   ```bash
+   $ sudo grep -q "io.containerd.kata.v2" "$cfg" || cat <<EOT | sudo tee -a "$cfg"
+   $ [plugins]
+     	[plugins.cri]
+     		[plugins.cri.containerd]
+     			no_pivot = false
+     			default_runtime_name = "kata"
+     		[plugins.cri.containerd.runtimes.kata]
+     			runtime_type = "io.containerd.kata.v2"
+     		[plugins.linux]
+     			shim = "/usr/bin/containerd-shim"
+     			shim_debug = true
```

Comment:
uhmm last time I tried to enable shim debug with this option, it didn't work, I add to add the following

```
[debug]
  level = "debug"
[plugins]
     	[plugins.cri]
..........
```

Reply:
Thanks - I'll double-check that tomorrow...

Reply:
I can confirm that this doesn't work for me, I need to add `[debug] level="debug"`

Reply:
@devimc - I don't get debug even with that :( What version of containerd are you using (I'm trying with v1.4.1).

Reply:
I'm using `containerd v1.3.2`. This is my config file

```toml
[debug]
  level = "debug"
[plugins]
  [plugins.cri]
    [plugins.cri.containerd]
      [plugins.cri.containerd.runtimes]
        [plugins.cri.containerd.runtimes.runc]
           runtime_type = "io.containerd.runc.v1"
           [plugins.cri.containerd.runtimes.runc.options]
             BinaryName = "/usr/bin/runc"
             Root = ""
        [plugins.cri.containerd.runtimes.kata]
           runtime_type = "io.containerd.kata.v2"
           privileged_without_host_devices = true
           pod_annotations = ["io.katacontainers.*"]
        [plugins.linux]
           shim = "/usr/bin/containerd-shim"
           shim_debug = true
```

Reply:
I happened to see this discussion in my email.  It would be easier for me if the config showed how to enable for all of the supported hypervisors. At the very least the instructions should show how to enable annotations like the example above shows.  Isn't `kata-deploy` becoming the recommended installation method?  There is a kata 2.0 rc0 on Docker hub for `kata-deploy`. To make this more generic, I would consider using that installation method.